extra-libs/med/spec

```diff
@@ -0,0 +1,5 @@
+VER=4.1.1
+SRCS="tbl::https://files.salome-platform.org/Salome/other/med-$VER.tar.gz"
+CHKSUMS="sha256::dc2b5d54ebf0666e3ff2e974041d2ab0da906061323537023ab165d573389dd0"
+# No CHKUPDATE here because its upstream now has no information about
+# this package, and it's not yet available on Anitya.
```

Comment:
Use "folder" or "custom" mode in Anitya.

Reply:
The problem is lack of update information.

Reply:
The folder itself is not browsable, and the website has now no info about this library, only their suites.